rmc_app_ui.py

```diff
@@ -0,0 +1,37 @@
+import os
+import redis
+import rmc_data_file
+
+from flask import Flask ,render_template 
+from flask.ext.bower import Bower
+from redis import Redis
+
+app=Flask(__name__)
+Bower(app)
+
+app.config.from_pyfile('config.cfg')
+
+
+r = redis.Redis(host='localhost', port=6379, db=0)
+
+content_dict={}
+content_dict['UPTIME'] ={'UPTIME':r.get('uptime')}
```

Comment:
Good start. It would be good to put all the keys in a Dicttionary so for example `key_mapping{'<local_key>:<redis_key>'}` This could go in a config file.

Then later you can simply iterate over the key's at and get the values from redis and update them in a context_dict